src/cross_section_measurement/make_control_plots_fromTrees.py

```diff
@@ -142,20 +168,34 @@ def make_plot( channel, x_axis_title, y_axis_title,
 
                           subtract = ['TTJet', 'V+Jets', 'SingleTop'] )
         # Normalise control region correctly
-        n_qcd_predicted_mc_signal = signal_region_hists['QCD'].Integral()
-        n_qcd_predicted_mc_control = control_region_hists['QCD'].Integral()
-        n_qcd_control_region = qcd_from_data.Integral()
+        nBins = signal_region_hists['QCD'].GetNbinsX()
+        n, error = signal_region_hists['QCD'].integral(0,nBins+1,error=True)
+        n_qcd_predicted_mc_signal = ufloat( n, error)
+
+        n, error = control_region_hists['QCD'].integral(0,nBins+1,error=True)
+        n_qcd_predicted_mc_control = ufloat( n, error)
+
+        n, error = qcd_from_data.integral(0,nBins+1,error=True)
+        n_qcd_control_region = ufloat( n, error)
+
+        print n_qcd_predicted_mc_signal
+        print n_qcd_predicted_mc_control
+        print n_qcd_control_region
```

Comment:
They are used in calculating the QCD scale factors?  I removed the print statements.